mixins/demo/provider-mixin.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
```

Comment:
Since the demo doesn't really show much visually and just duplicates a lot of the awesome stuff from the README, I think it could be excluded. We can rely on unit tests for a lot of this as well.

Reply:
Yeah admittedly I made it mostly to make sure that the code I wrote for the README is actually valid 😆 But yeah, not a very interesting demo page for sure.